custom_components/frigate/const.py

```diff
@@ -38,6 +38,7 @@
 CONF_CAMERA_STATIC_IMAGE_HEIGHT = "camera_image_height"
 CONF_NOTIFICATION_PROXY_ENABLE = "notification_proxy_enable"
 CONF_RTMP_URL_TEMPLATE = "rtmp_url_template"
+CONF_EXPIRE_NOTIFICATIONS_AFTER_MINS = "expire_notifications_after_mins"
```

Comment:
Maybe `notification_proxy_expire_after_mins` for naming consistency with CONF_NOTIFICATION_PROXY_ENABLE?